NEWS.rst

```diff
@@ -3,6 +3,112 @@ http://twistedmatrix.com/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 21.7.0.rc1 (2021-07-10)
+===============================
+
+Features
+--------
+
+- support py3.10b3 (by fixing twisted.trial.unittest.TestCase.flushWarnings) (#10111)
```

Comment:
```suggestion
- support py3.10b4 (by fixing twisted.trial.unittest.TestCase.flushWarnings) (#10111)
```

Reply:
Thanks.  I will update.

In practice this release supports  3.10b4...but I think that the ticket number should 10224 and not 10111.

The changes for ticket 10224 will not be part of this release, but since no changes are required between 3.10b3 an 3.10.b4 I think that is safe to advertise that 3.10b4 is supported.

Reply:
seems good to me

Reply:
The applied suggestion + the comment from Glyph over the mailing list
```suggestion
- Python 3.10b3 is now supported (#10224)
- Type hinting was added to twisted.internet.defer, making this is the first release
  of Twisted where you might reasonably be able to use mypy without your own custom
  stub files (#10017) 
```